transaction/restore-transaction.go

```diff
@@ -42,16 +69,24 @@ func RestoreTransactionFx(w http.ResponseWriter, r *http.Request) (string, inter
 		return "", nil, err
 	}
 
-	err = lib.SetFirestoreErr(models.TransactionsCollection, transaction.Uid, transaction)
+	err = saveTransactionToDb(collection, *transaction)
```

Comment:
we have a `common.SaveTransactionsToDB` function. Should/could we use it?
Something like:
```
if err = common.SaveTransactionsToDB([]models.Transaction{*transaction}, collection); err != nil {
   ...
}
```